code/modules/modular_computers/file_system/programs/command/comm.dm

```diff
@@ -301,19 +301,19 @@ var/last_message_id = 0
 	if(!frequency) return
 
 	var/datum/signal/status_signal = new
-	status_signal.source = src
+	status_signal.source = UNLINT(src)
```

Comment:
src is meaningless in global procs

Reply:
Not sure what you mean, would the code down frequency.post_signal() needs it ?

Reply:
src isn't a thing in global procs, you might mean usr?

Reply:
I see, I checked agaisn't BayStation12 repo, That's true, I don't find any src there. Question resolved